### PR TITLE
Update OCP NIST CPE and add installed_app_is_ocp3 OVAL check

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

**Comment by mpreisler** [comment]
The jenkins fail is legitimate here, **do not merge** until it's fixed.

**Comment by redhatrises** [comment]
> The jenkins fail is legitimate here, **do not merge** until it's fixed.

Thanks @mpreisler. Hadn't seen that the build failed on this. No one was planning on merging it.

**Comment by jan-cerny** [comment]
@shawndwells Their website is still down, but I have downloaded an older version (2018-08-29) of the NIST CPE dictionary. on my harddrive. I have checked it and `cpe:/a:redhat:openshift_container_platform:3.10` is there. `cpe:/a:redhat:openshift_container_platform:3.11` isn't there, but I would expect it will be eventually added. Therefore I believe the change proposed by @redhatrises is correct.

File: shared/checks/oval/installed_app_is_ocp3.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      source="CPE" />
+      <description>The application installed installed on the system is
+      OpenShift 3.</description>
+    </metadata>
+    <criteria>
+      <extend_definition comment="RHEL7 OS installed" definition_ref="installed_OS_is_rhel7" />
+      <criterion comment="OpenShift Node is installed" test_ref="test_ocp3_node" />
+      <criterion comment="OpenShift Hyperkube is installed" test_ref="test_ocp3_hyperkube" />
+      <criterion comment="Atomic OpenShift is installed" test_ref="test_ocp3_atomic" />
+    </criteria>
```
Comment (jan-cerny) [inline shared/checks/oval/installed_app_is_ocp3.xml]: Does it mean that all 3 things have to be installed at the same time?

Comment (redhatrises) [reply shared/checks/oval/installed_app_is_ocp3.xml]: @jan-cerny yes

File: ocp3/product.yml [hunk 2]
```diff
@@ -11,5 +11,5 @@ pkg_system: "rpm"
 init_system: "systemd"
 
 cpes:
-  - "cpe:/a:redhat:openshift:3.0"
-  - "cpe:/a:redhat:openshift:3.1"
+  - "cpe:/a:redhat:openshift_container_platform:3.10"
```
[the remark this answers is not in the document]
Comment (redhatrises) [reply ocp3/product.yml]: @jan-cerny done

**Comment by shawndwells** [comment]
@redhatrises - after this was merged, all checks now showup as notapplicable

e.g
image

**Pull request merged**